app/api/ddi-index-api/cdos.js

```diff
@@ -0,0 +1,157 @@
+const { get } = require('./base')
+const { formatToGds } = require('../../lib/date-helpers')
+
+const cdosEndpoint = 'cdos'
+
+const options = {
+  json: true
+}
+/**
+ * @typedef SummaryPersonDto
+ * @property {number} id - e.g. 10,
+ * @property {string} firstName - e.g. 'Scott',
+ * @property {string} lastName - e.g. 'Pilgrim',
+ * @property {string} personReference - e.g. 'P-1234-5678'
+ */
+/**
+ * @typedef SummaryDogDto
+ * @property {number} id - e.g. 300013,
+ * @property {string} dogReference - e.g. 'ED300013',
+ * @property {string} status - e.g. 'Pre-exempt'
+ */
+/**
+ * @typedef SummaryExemptionDto
+ * @property {string} policeForce - e.g. 'Cheshire Constabulary',
+ * @property {string} cdoExpiry - e.g. '2024-03-01'
+ * @property {string} joinedExemptionScheme - e.g. '2024-03-01'
+ */
+/**
+ * @typedef SummaryCdoDto
+ * @property {SummaryPersonDto} person
+ * @property {SummaryDogDto} dog
+ * @property {SummaryExemptionDto} exemption
+ */
+
+/**
+ * @typedef SummaryCdo
+ * @property {number} id - e.g. 300013,
+ * @property {string} index - e.g. 'ED300013',
+ * @property {string} status - e.g. 'Pre-exempt',
+ * @property {string} owner - e.g. 'Scott Pilgrim',
+ * @property {string} personReference - e.g. 'P-1234-5678',
+ * @property {Date|null} cdoExpiry - e.g. 2024-03-01,
+ * @property {string} humanReadableCdoExpiry - e.g. 2024-03-01,
+ * @property {Date|null} joinedExemptionScheme - e.g. 2024-03-01,
+ * @property {string} policeForce - e.g. 'Cheshire Constabulary'
+ */
+
+/**
+ * @param {SummaryCdoDto} summaryCdo
+ * @return {SummaryCdo}
+ */
+const summaryCdoMapper = (summaryCdo) => {
+  const owner = [
+    summaryCdo.person.firstName,
+    summaryCdo.person.lastName
+  ].join(' ')
+
+  const cdoExpiryDto = summaryCdo.exemption.cdoExpiry ?? null
+  const joinedExemptionSchemeDto = summaryCdo.exemption.joinedExemptionScheme ?? null
+  const cdoExpiry = cdoExpiryDto !== null ? new Date(summaryCdo.exemption.cdoExpiry) : null
+
+  return {
+    id: summaryCdo.dog.id,
+    index: summaryCdo.dog.dogReference,
```

Comment:
I suggest not changing things as would be a bit of a pain, but for consistency, we should always refer to the dog index number as 'indexNumber', not dog reference